Dockerfile

```diff
@@ -30,4 +30,9 @@ COPY --from=build /home/builder/autotest-runner.jar ./
 # Copy cached dependencies
 COPY --from=build /home/gradle /home/gradle
 
+RUN pkill -f '.*GradleDaemon.*'
```

Comment:
There is one stopped `gradle` daemon in the image, those two lines remove it (and the time spent to look it up)